src/components/Management.tsx

```diff
@@ -0,0 +1,77 @@
+import { MANAGEMENT_STATUS } from '@/constants';
+import { format } from 'date-fns';
+import React from 'react';
+import { AdvertisementInterface } from 'request';
+import styled from 'styled-components';
+import ManagementForm from './ManagementForm';
+import ManagementHeader from './ManagementHeader';
+
+interface ManagementProps {
+  advertisements: AdvertisementInterface[];
+}
+
+const Management = ({ advertisements }: ManagementProps) => {
+  const DATE_FORMAT = 'yyyy-MM-dd';
+  return (
+    <Wrapper>
+      <ManagementHeader />
+      <FormContainer>
+        {advertisements.map((advertisement) => {
+          const startDate = format(
+            new Date(advertisement.startDate),
+            DATE_FORMAT
+          );
+
+          const propsAdverTisement = {
+            title: advertisement.title,
+            status: MANAGEMENT_STATUS[advertisement.status],
+            startDate,
```

Comment:
요것도 통일성있게 이름을 추가해주면 좋을 것 같아요!!
```suggestion
            startDate: startDate,
```